### PR TITLE
completed html forms

### PR DESCRIPTION
**Pull request by Boyul-Kim**
image

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Boyul! Your `label` elements are lacking a value in their `for` attributes to correlate to `id`s on the target form control elements. Additionally, you are missing values in the `id`s on your form control elements. Please add a `for` attribute on each label that corresponds to an `id`on a form control element.

**Comment by Boyul-Kim** [comment]
Hey Cody
Thanks for pointing those out!

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        <option value="selectAnOption">Select an option</option>
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+    </div>
+
+    <label for="radio">Favorite Language (choose one)</label>
+    <div id="radio">
+      <input type="radio" value="HTML"> HTML
```
Comment (codingcodymiller) [inline html-forms/index.html:28]: For radio inputs, the `label` should surround the text describing the radio input's value, which would here be the text "HTML". Make sure to give each label a `for` attribute and an individual `id` for each radio input 👍

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Good improvement! I still have a few more notes for you:

**Comment by Boyul-Kim** [comment]
corrected!

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**